### PR TITLE
Logout

### LINKED ISSUE #31: Add logout button in /profile
Add logout button in /profile, this would mean creating a logout servlet and api

### PR DESCRIPTION
**Pull request by San7o**
Added /logout and a logout link in the profile page. Closes #31

### CODE REVIEW

File: src/main/java/Logout.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import java.io.*;
+
+import javax.servlet.http.HttpSession;
+import javax.servlet.http.*;
+import javax.servlet.*;
+
+import java.util.HashMap;
+
+// This servlet handles Logout, invalidating the 
+// sessione and redirecting to the login page
```
Comment (Qw3rty26) [inline src/main/java/Logout.java:8]: Little typo in line 10, typed "sessione" instead of "session"

File: src/main/java/Logout.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import java.io.*;
+
+import javax.servlet.http.HttpSession;
+import javax.servlet.http.*;
+import javax.servlet.*;
+
+import java.util.HashMap;
```
Comment (Qw3rty26) [inline src/main/java/Logout.java]: Do we really need to import `java.util.HashMap` in Logout.java servlet? (Line 7)

File: src/main/webapp/WEB-INF/web.xml [hunk 3]
```diff
@@ -34,21 +33,29 @@
    -->
    <servlet>
         <servlet-name>hello_servlet</servlet-name>
-	<servlet-class>HelloServlet</servlet-class>
+	    <servlet-class>HelloServlet</servlet-class>
```
Comment (Qw3rty26) [inline src/main/webapp/WEB-INF/web.xml]: Indentation in line 36, 41 and 46 differ from the one used in lines 51 and 56. Please choose either one (I'd suggest using indentation like the one used in line 51) and keep it the same for the rest of the code to make reading it simpler and faster.

**Review by Qw3rty26 — CHANGES_REQUESTED** [review]
Looking good. Just a little question about the need of importing `java.util.HashMap` inside the `Logout.java` servlet at line 7 and a few inconsistencies with the indentation inside the `web.xml`. Other than that this looks very good.

**Comment by San7o** [comment]
Fixed code

**Review by Qw3rty26 — APPROVED** [review]
Looks good. Typo still present in line 10 of Logout.java....Nonetheless good to go.

**Pull request merged**